src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -130,6 +130,9 @@ private VariantOverlapAnnotator initializeOverlapAnnotator(final FeatureInput<Va
             }
         }
 
+        if (descriptions.contains(null)){
+            throw new IllegalStateException("getVCFAnnotationDescriptions should not contain null");
```

Comment:
Make it clear in this error message that the error is due to an incorrect implementation of `getDescriptions()` in one or more of the annotation classes